src/main/java/com/arpnetworking/metrics/impl/BaseFileSink.java

```diff
@@ -414,5 +453,7 @@ protected void validate(final List<String> failures) {
         private static final Boolean DEFAULT_PRUDENT = Boolean.FALSE;
         private static final Boolean DEFAULT_DROP_WHEN_QUEUE_FULL = Boolean.FALSE;
         private static final Integer DEFAULT_MAX_QUEUE_SIZE = Integer.valueOf(500);
+        private static final String DEFAULT_MAX_FILE_SIZE = "100MB";
+        private static final String DEFAULT_TOTAL_SIZE_CAP = "20GB";
```

Comment:
So this may be a difference between how Logback and LogackSteno implemented it. I would have to check but my understanding was that max history limits the number of files not the number of periods. Then max size limits the size of each file. 

Can you confirm which behavior Logback and Steno each use? If it is as you say that max history only applies to time periods then we do indeed have a problem; that max size cannot limit total size effectively on its own.